Description: do not pass wrong -L flag when linking with libqbscore
Author: Adam Majer <adamm@zombino.com>
Forwarded: not-needed
Last-Update: 2015-03-13

--- a/src/lib/corelib/use_installed_corelib.pri
+++ b/src/lib/corelib/use_installed_corelib.pri
@@ -4,11 +4,9 @@
 
 QBSLIBDIR=$${PWD}/../../lib
 unix {
-    LIBS += -L$$QBSLIBDIR -lqbscore
+    LIBS += -lqbscore
 }
 
-!qbs_disable_rpath:unix:QMAKE_LFLAGS += -Wl,-rpath,$${QBSLIBDIR}
-
 !CONFIG(static, static|shared) {
     QBSCORELIBSUFFIX = $$QBS_VERSION_MAJ
 }
